samples/suit/smp_transfer/README.rst

```diff
@@ -0,0 +1,382 @@
+:orphan:
+
+.. _nrf54h_suit_sample:
+
+SUIT DFU: Firmware update on the nRF54H20 SoC
```

Comment:
```suggestion
SUIT: Device firmware update on the nRF54H20 SoC
```
To me, SUIT itself is about DFU, DFU next to it is also about DFU, and then we have "firmware update" right after it. This is a lot of firmware updates in one title.